pkg/apis/extensions/validation/validation_test.go

```diff
@@ -33,6 +33,33 @@ import (
 	psputil "k8s.io/kubernetes/pkg/security/podsecuritypolicy/util"
 )
 
+func validPSP() *extensions.PodSecurityPolicy {
```

Comment:
Could you put this function at the end of a file?

Reply:
Sure can!